debian/control

```diff
@@ -0,0 +1,13 @@
+Source: on-static-common
```

Comment:
IMO "builder" is inappropriate for a deb. on-static-common is inherited from Onrack and "common" means there're specific static files too. So I think it's a proper name.